docs/pages/database-access/auto-user-provisioning/postgres.mdx

```diff
@@ -90,6 +94,78 @@ Users created within the database will:
 
 (!docs/pages/includes/database-access/auto-user-provisioning/postgres15-grant-create.mdx!)
 
+### User does not have CONNECT privilege error
+
+You may encounter the following error when the admin user or the roles assigned
+to the auto-provisioned user do not have the permission to connect to the target
+database:
+
+```shell
+> tsh db connect mysql --db-name <database> example
+...
+FATAL: permission denied for database "<database>"
+DETAIL: User does not have CONNECT privilege.
+```
+
+Make sure `CONNECT` is granted to the admin user and the respective roles:
+```sql
+GRANT CONNECT ON DATABASE <database> to "teleport-admin";
+GRANT CONNECT ON DATABASE <database> to "reader";
+```
+
+### Cannot execute in a read-only transaction error
+
+You may encounter the following error when connecting to an AWS RDS Aurora
+reader endpoint:
+```shell
+> tsh db connect mysql --db-name <database> example
+psql: error: connection to server at "localhost" (::1), port 12345 failed: Connection refused
+...
+ERROR: cannot execute CREATE ROLE in a read-only transaction (SQLSTATE 25006)
+```
+
+Database auto-user provisioning is curerntly not compatible with RDS Aurora
+reader endpoints. Please use auto-user provisioning on the primary endpoints.
+
+### No schema has been selected error
+
+You may encounter the following error when connecting when the admin user does not have `USAGE`
```

Comment:
```suggestion
You may encounter the following error when connecting if the admin user does not have `USAGE` and `CREATE`
```

Reply:
This error message happens only when `USAGE` is not granted. Without `CREATE`, the error message is more obvious. Will fix the `when` -> `if`